templates/zerver/api/get-presence.md

```diff
@@ -0,0 +1,59 @@
+# Get user presence
+
+Get the presence status for a specific user.
```

Comment:
We should talk about the use case here, namely that you can use this for things like an employee directory integration to show a given user's Zulip presence status.